libs/design/src/atoms/form/checkbox/checkbox.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, OnInit, Input, HostBinding, ChangeDetectionStrategy, forwardRef } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+
+let checkboxIdNum = 0;
+
+@Component({
+  selector: 'daff-checkbox',
+  templateUrl: './checkbox.component.html',
+  styleUrls: ['./checkbox.component.scss'],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => DaffCheckboxComponent),
+      multi: true
+    }
+  ],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DaffCheckboxComponent implements ControlValueAccessor {
+  @Input() value: any;
+  @Input() checked = false;
```

Comment:
Need these additional inputs.
`name`
`aria-label`
`aria-labelledby`

Reply:
Okay added this as well as tests in c2611c67bf5504a3d3d15b9c508963809e07b628. Let me know if that looks good to you.